gui-wm/dwl/dwl-9999-r1.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 2022-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit savedconfig toolchain-funcs
+
+WLROOTS_SLOT="0/16"
+if [[ ${PV} == *9999* ]]; then
+	EGIT_REPO_URI="https://github.com/djpohly/dwl"
+	inherit git-r3
+
+	# 9999-r0: main (latest stable wlroots release)
+	# 9999-r1: wlroots-next (wlroots-9999)
+	case ${PVR} in
+		9999)
+			EGIT_BRANCH=main
+			;;
+		9999-r1)
+			EGIT_BRANCH=wlroots-next
+			WLROOTS_SLOT="0/9999"
+			;;
+	esac
+else
+	SRC_URI="https://github.com/djpohly/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
If you're the maintainer, pretty please consider making real dist tarballs (possibly with the lord's xz compression)? github tarballs aren't reliable and we should avoid relying on them when we have a choice.

... although I see it's using homebrew Makefiles, not autotools or Meson, so I guess that's tricky for now.